crates/compiler/can/src/expr.rs

```diff
@@ -39,7 +39,8 @@ pub type PendingDerives = VecMap<Symbol, (Type, Vec<Loc<Symbol>>)>;
 #[derive(Clone, Default, Debug)]
 pub struct Output {
     pub references: References,
-    pub tail_call: Option<Symbol>,
+    pub early_tail_calls: Vec<Option<Symbol>>,
+    pub final_tail_call: Option<Symbol>,
```

Comment:
Why does this distinction matter? Why not have `tail_calls: Vec<Symbol>`?

Reply:
I couldn't get things working initially, but they seem to be working now. Good thing you asked!